app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<p>
+  Driver ID: <%= @driver.id %>
+</p>
+<p>
+  Driver name: <%= @driver.name %>
+</p>
+<p>
+  Driver VIN: <%= @driver.vin %>
+</p>
+<p>
+  Total Driver Earnings: <%= @driver.total_earnings %>
+</p>
+<p>
+  Driver's Average Rating: <%= @driver.average_rating %>
+</p>
+<p>
+  Driver's Trips:
```

Comment:
I'm not sure what happened with this section, but it ends up being really difficult to read. Recommend creating an overall semantic section of some sort of encapsulate all of this trip information. As well as possibly another HTML container that would contain the details of one piece of trip info.